staging/src/k8s.io/client-go/pkg/api/v1/generated.proto

```diff
@@ -1413,6 +1413,12 @@ message LoadBalancerStatus {
   repeated LoadBalancerIngress ingress = 1;
 }
 
+// LocalFsVolume represents a filesystem-based local volume
```

Comment:
Yep. This file is copied from k8s.io/kubernetes/pkg/api/v1/generated.proto. If they are out of sync, verification-all.sh will fail.